rclcpp/src/rclcpp/time_source.cpp

```diff
@@ -260,6 +278,7 @@ void TimeSource::destroy_clock_sub()
 
 void TimeSource::on_parameter_event(const rcl_interfaces::msg::ParameterEvent::SharedPtr event)
 {
+  RCLCPP_DEBUG(logger_, "this node should not get others' parameter event if CFT is supported");
```

Comment:
i believe this message should be printed only when CFT is supported as implementation. currently this debug information will come up every time if implementation does not support CFT. and it introduces complication as debug information. and i think if we can use CFT, this message is warning, since this is not what we expects if CFT works precisely. or filtering on writer is dependent on implementation, so we would probably keep this as debug information.